BrickController2/BrickController2/UI/Pages/ControllerProfilePage.xaml

```diff
@@ -67,6 +68,8 @@
                                     </Grid.ColumnDefinitions>
                                     <controls:ColorImage Grid.Column="0" Icon="{Binding ControllerEvent.EventType, Converter={StaticResource EventTypeToImage}}" Color="{DynamicResource PrimaryColor}" WidthRequest="30" HeightRequest="30" VerticalOptions="Center"/>
                                     <StackLayout Grid.Column="1" Orientation="Horizontal" VerticalOptions="Center">
+                                        <Label Text="{Binding ControllerEvent.ControllerId}" FontSize="Large" FontAttributes="Bold" IsVisible="{Binding ControllerEvent.ControllerId, Converter={StaticResource IsStringNotNullOrEmptyConverter}}"/>
+                                        <Label Text=" - " FontSize="Large" FontAttributes="Bold" IsVisible="{Binding ControllerEvent.ControllerId, Converter={StaticResource IsStringNotNullOrEmptyConverter}}"/>
```

Comment:
Nit pick: I vote for removal of BOLD attribute here.

Reply:
It's done:
https://github.com/vicocz/brickcontroller2/pull/75/commits/f3be1fc7d1d2219ef83205a47b7102224b3d73bf